### DISCUSSION TITLE
Slack Alerts are not coming properly for Dimension

### DISCUSSION THREAD

**Discussion by jithendra945** [cuebook/CueObserve#94] [Ideas]
Hi, when I run an anomaly definition with a dimension given when I keep Top 10, 
From those 10 Categories/dimensions, only 1 Alert is coming, but it should be 10 Alerts na, for each Category.....

**Comment by sachinkbansal** [comment]
It sends max of 1 alert per anomaly definition. This is to reduce the number of alerts.
However, the slack alert does mention the number of anomalies.

**Reply by jithendra945** [reply]
It is mentioned only, no.of anomalies Published. 
Also, out of 10 categories in dimension, is it sending the 1 with high Filter contribution ?

How it is Picking the one thing, out of 10, on what Basis ?

**Reply by sachinkbansal** [reply]
Yes, it picks the filter with the max contribution.
